cdc/log.cc

```diff
@@ -234,7 +237,29 @@ static schema_ptr create_log_schema(const schema& s, std::optional<utils::UUID>
         for (const auto& column : columns) {
             auto type = column.type;
             if (is_data_col) {
-                type = tuple_type_impl::get_instance({ /* op */ data_type_for<column_op_native_type>(), /* value */ type, /* ttl */long_type});
+                type = visit(*type, make_visitor(
+                    // lists are represented as map<timeuuid, value_type>. Otherwise we cannot express delta
+                    [] (const list_type_impl& type) {
+                        return map_type_impl::get_instance(type.name_comparator(), type.value_comparator(), false);
```

Comment:
no need to freeze() here?

Reply:
Last "false" says it is a frozen map (not multicell).